### ISSUE TITLE
Unable to access https://mxnet.incubator.apache.org/ in China, got 404 response.

### ISSUE THREAD

**Issue by blessalanou** [apache/incubator-mxnet#16291]

**Comment by lanking520** [comment]
@blessalanou I am sorry to hear that. It might be some blocker in the Network in China. Everything is fine on the US side. @aaronmarkham is there anything we can help here?

**Comment by aaronmarkham** [comment]
There's another issue with jquery being blocked by the great firewall. I was going to move the libs locally so we can avoid that, but if they're blocking the whole site, there's not much we can do. (AFAIK)

**Comment by ptrendx** [comment]
Actually I have a similar issue and I'm in US - https://mxnet.incubator.apache.org does not work for me, while https://mxnet.apache.org does.

**Comment by blessalanou** [comment]
Thank you guys. https://mxnet.incubator.apache.org works OK.
